code/modules/client/preferences.dm

```diff
@@ -137,11 +137,11 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/mute_xeno_health_alert_messages = TRUE
 
 	///whether the user wants to hear tts
-	var/sound_tts = TRUE
+	var/sound_tts = TTS_SOUND_ENABLED
```

Comment:
the pref no longer works. You can't disable TTS at all.

Reply:
also changing tts volume to 0 doesn't work, it won't save (I assume its the same issue). Although you can set it to 1 fine

Reply:
latter is intended